chpl/chpl-service/src/main/java/gov/healthit/chpl/questionableactivity/ListingQuestionableActivityProvider.java

```diff
@@ -588,6 +620,28 @@ private boolean isListingRealWorldTestingEligible(Long listingId) {
         return rwtEligServiceFactory.getInstance().getRwtEligibilityYearForListing(listingId, LOGGER).getEligibilityYear().isPresent();
     }
 
+    private boolean hasPromotingInteroperabilityHistoryChanged(CertifiedProductSearchDetails existingListing,
+            CertifiedProductSearchDetails updatedListing) {
+        existingListing.getPromotingInteroperabilityUserHistory().sort(new PromotingInteroperabilityComparator());
```

Comment:
Does this need a sort? If not, we could remove the comparator too, right?

Reply:
RIght!